internal/configuration.go

```diff
@@ -0,0 +1,17 @@
+package internal
+
+import "github.com/caarlos0/env/v11"
+
+type Configuration struct {
+	GithubToken string `env:"GITHUB_TOKEN,notEmpty"`
+	OpenAIToken string `env:"OPENAI_API_KEY,notEmpty"`
+}
+
+func LoadConfiguration() (Configuration, error) {
```

Comment:
Consider handling missing environment variables more gracefully. Instead of using 'notEmpty' which will throw an error, you might want to check for the presence of these env vars explicitly and provide a more informative error message.